src/ui/common/src/components/tables/PaginatedSearchTable.tsx

```diff
@@ -225,6 +226,7 @@ export const PaginatedSearchTable: React.FC<PaginatedSearchTableProps> = ({
                                 columnIndex < columns.length - 1
                                   ? '1px solid rgba(224, 224, 224, 1);'
                                   : 'none',
+                              fontSize: '16px', // This is needed for consistency.
```

Comment:
@agiron123 is there a single place we can track all font sizes? It would make fonts easier to track and less effort keeping them consistent

Reply:
Yes, theme.tsx should be the place to put this. Ideally we should be using units throughout our project. (and even add things like fontWeight to our theme file).

I am pretty guilty of just using pixel values, when I should be using our theme's unit system instead.

I've filed a task here to address us not using units everywhere in our project:
https://linear.app/aqueducthq/issue/ENG-2236/use-theme-constants-throughout-frontend-project